src/XR/features/WebXRNearInteraction.ts

```diff
@@ -791,23 +786,33 @@ export class WebXRNearInteraction extends WebXRAbstractFeature {
         const easingFunction = new QuadraticEase();
         easingFunction.setEasingMode(EasingFunction.EASINGMODE_EASEINOUT);
 
+        const hoverSizeVec = new Vector3(this._controllerPickRadius, this._controllerPickRadius, this._controllerPickRadius);

```

Comment:
All the numbers sounds a bit magical, wondering if we should comment on their provenance ?

Reply:
I made the fractions based off of the float values that were in place before, I'll add in descriptors though.